.github/workflows/create-release.yml

```diff
@@ -35,3 +37,197 @@ jobs:
       - run: |
           echo "Saw Liquibase version ${{ steps.collect-data.outputs.liquibaseVersion }}"
           echo "Saw Extension version ${{ steps.collect-data.outputs.liquibaseVersion }}"
```

Comment:
Should this be `${{ steps.collect-data.outputs.extensionVersion }}`?

Reply:
Yep, will fix that